juniper_official/chassis/check-fan-state-rpm.rule

```diff
@@ -81,17 +81,9 @@ healthbot {
                 }
                 type integer;
                 description "RPM percentage";
-                plots rpm-plots {
+                plots rpm {
                     unit percentage;
                     triggers rpm-percent;
-                    thresholds high-threshold {
```

Comment:
Do not remove all the thresholds.
Threshold should be removed for fec-uncorrected plot.